.gitignore

```diff
@@ -8,7 +8,6 @@ __pycache__/
 
 # Distribution / packaging
 .Python
-build/
```

Comment:
I assume we no longer want to ignore 'build' as we have a few files in build that are needed for building container images